src/widgets/dialogs/BadgePickerDialog.cpp

```diff
@@ -0,0 +1,76 @@
+#include "BadgePickerDialog.hpp"
+#include <QSizePolicy>
+#include "singletons/Resources.hpp"
+
+#include "providers/twitch/TwitchBadges.hpp"
+
+namespace chatterino {
+
+BadgePickerDialog::BadgePickerDialog(QList<DisplayBadge> badges,
+                                     QWidget *parent)
+    : QDialog(parent)
+{
+    this->dropdown_ = new QComboBox;
+    auto vbox = new QVBoxLayout(this);
+    auto buttonBox = new QHBoxLayout;
+    auto okButton = new QPushButton("OK");
+    auto cancelButton = new QPushButton("Cancel");
```

Comment:
Use https://doc.qt.io/qt-5/qdialogbuttonbox.html instead of a 'home made' layout/QPushButtons

Reply:
Fixed in [`e63110e` (#1704)](https://github.com/Chatterino/chatterino2/pull/1704/commits/e63110ef2fda3c8b4e352cca52b35e18748a044b)